enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,77 @@
 const enhancer = require('./enhancer.js');
 // test away!
+
+it('item => new item with durability === 100', () => {
```

Comment:
for this size app this isn't wrong, but it's always a good idea to wrap tests dealing with the same function in the same `describe` block, and arguably even all function `describe` blocks should be wrapped in a single `describe` block for the file they're all imported from to be more readable and keep things separated--check how Luis did it in the guided demo today to see an example